dashboard/src/components/CatalogList/CatalogList.test.tsx

```diff
@@ -8,7 +8,7 @@ import { CardGrid } from "../Card";
 import { MessageAlert } from "../ErrorAlert";
 import PageHeader from "../PageHeader";
 import SearchFilter from "../SearchFilter";
-import ChartList from "./ChartList";
+import CatalogList from "./CatalogList";
```

Comment:
I'd rename this to actually AppsCatalog so we do not overload `Catalog` and leave the door open for other kind of catalogs :)

Reply:
@migmartri the idea is that eventually we will include the ServiceCatalog classes in the Catalog view as well, right? In that case, it won't make sense for it to be called AppsCatalog, but I don't mind calling it that for now and then changing it when we get to that point.

Reply:
That's a good point. Catalog is fine then. Thanks!